src/main/js/Upload/FileUploadButton.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import React from "react";
+import type { Branch } from "@scm-manager/ui-types";
+import { translate } from "react-i18next";
+import injectSheet from "react-jss";
+import classNames from "classnames";
+import { Link } from "react-router-dom";
+
+const styles = {
```

Comment:
Do we really have to set the colors here? Wouldn't like to see color codes distributed in core and multiple plugins. Would be a hell whenever we would like to change them.

Reply:
done